### ISSUE TITLE
SCAMMERS

### ISSUE THREAD

**Issue by Baabiidawn** [ton-blockchain/wallet-contract-v5#53]
They are pretending to be the dogs app on telegram and scammed me out of my TON 
YOUR JUDGMENT DAY WILL COME YOU HEATHENS
![IMG_6475](https://github.com/user-attachments/assets/39a13ac6-e4dc-4f74-8617-69a1990e4eba)
![IMG_6474](https://github.com/user-attachments/assets/fbe8cfa7-7dcc-4455-8318-25216ff75570)
![IMG_6476](https://github.com/user-attachments/assets/90f33689-9503-47a4-a003-7b2d1942d872)
![IMG_6479](https://github.com/user-attachments/assets/480dfe64-a83d-49e9-9cf9-4f1ebfc0c830)

**Comment by latan22** [comment]
Yes they have also scammed my ton

**Comment by Ashkan-1989** [comment]
They also stolen my tons,those can be returned to me or not?

**Comment by Debojit8099** [comment]
Yes please solve this problem they have also scammed my toon , please return my toon 😔
